### PR TITLE
PAAS-2971: split v41 migration into 3 migration packages

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-2971

Short description:

**Don't forget** to update the README and any related documentation if needed: https://confluence.jahia.com/x/bYATAg

### CODE REVIEW

**Review by kevinvandel — CHANGES_REQUESTED** [review]
As said in the ticket:
> don't forget to update the "env" version number in the Jahia install package, it should be 43 in the end

Also, we can remove the "eventsUpdate" in packages 42 and 43, but if we do so I fear we forget to add is back in the next migration package...

**Comment by Ronan-Gorain** [comment]
Should be ok now

File: packages/jahia/migrations/migrate-to-v42.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            jps: "${baseUrl}../redeploy-galera-nodes.yml"
+    - else:
+        - setGlobals:
+            targetDockerTag: 10.11.7-almalinux-9
+        - getGaleraNodeNameIndex: ${nodes.sqldb.first.id}
+        - muteDatadogHost:
+            target: "*"
+            duration: 60
+        - muteEnvWideMonitors
+        - stopJahia: cp, proc
```
Comment (kevinvandel) [inline packages/jahia/migrations/migrate-to-v42.yaml:138]: Is it normal that we stop but never seem to restart?

**Review by kevinvandel — CHANGES_REQUESTED** [review]

File: packages/jahia/migrations/migrate-to-v42.yaml (hunk 1 above)
Comment (kevinvandel) [reply packages/jahia/migrations/migrate-to-v42.yaml:138]: I just tested it, the package fails because tomcat isn't restarted

**Review by kevinvandel — APPROVED** [review]

**Pull request merged**